tests/app/service_invite/test_service_invite_rest.py

```diff
@@ -274,10 +274,14 @@ def test_update_invited_user_for_invalid_data_returns_400(client, sample_invited
     assert response.status_code == 400
 
 
-def test_validate_invitation_token_returns_200_when_token_valid(client, sample_invited_user):
+@pytest.mark.parametrize('endpoint_format_str', [
+    '/invite/service/{}',
+    '/invite/check/service/{}',
```

Comment:
Minor: the "check" bit would be easier to interpret for me if it was on the end i.e. `/invite/service/{}/check`. I think that's because the first part of the path is scoping the resource "invite / service", while "check" is the action we're performing on it.

Reply:
hmmm, i'm not sure it should be on the end, largely because the token is a long variable length string and hard to find the end of (compared to a uuid which is fixed length). i think it's easy to miss detail like `/check` if it's at the end (like the `/edit` at the end of a google doc link. but that's just me. thoughts?

Reply:
we could have something like `check_token` in the URL to make it super explicit that it's a token, not a uuid, if that helps 🤔

Reply:
> the token is a long variable length string and hard to find the end of 

That's fair. How about a compromise: `/invite/service/check/{}`?